perfkitbenchmarker/beam_pipeline_options.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2017 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import yaml
+
+from perfkitbenchmarker import errors
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import kubernetes_helper
+
+FLAGS = flags.FLAGS
+
+
+def GetStaticPipelineOptions(options_list):
```

Comment:
A one line documentation, if possible ? Also I am curious about the options_list data type, so if you have an example feel free to add it to the documentation.